code/game/objects/structures/grille.dm

```diff
@@ -365,6 +367,19 @@
 /obj/structure/grille/get_dumping_location()
 	return null
 
+/obj/structure/grille/proc/temporary_shatter(time_to_go = 0 SECONDS, time_to_return = 4 SECONDS)
```

Comment:
I wish we had status effects for objects

Reply:
that'd be epic